collections/videoCollection.js

```diff
@@ -0,0 +1,21 @@
+var VideoCollection = Backbone.Collection.extend({
+  url: 'https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&q=brian-david-gilbert&type=video&videoEmbeddable=true&key=' + apiKey,
+
+  model: VideoModel,
+
+  parse: function (response) {
+    return response.items.map( function (video) {
+      return {
+        title: video.snippet.title,
+        description: video.snippet.description,
+        thumbnail: video.snippet.thumbnails.default.url,
+        videoId: video.id.videoId
+      }
+    })
+  },
+  
+  searchYoutube: function (searchQuery) {
+    this.url = 'https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&q=' + searchQuery + '&type=video&videoEmbeddable=true&key=' + apiKey;
```

Comment:
This is fine, but it would be better if you could just update the url then listen for changes on it and call fetch when it changes.